### PR TITLE
Fix NullPointerException (#1860)

### LINKED ISSUE #1860: NPE when servletContext is null
## Describe the bug
There is a `NullPointerException` when `servletContext` is `null` in `co.elastic.apm.agent.servlet.helper.ServletTransactionCreationHelper#getClassloader` (happens on standalone Spark)

## Steps to reproduce
Instrument a standalone Spark application (together with #1858), e.g.
```
public class Main {
    public static void main(String[] args() {
        get("/foo", (req, res) -> null);
    }
}
```

## Expected behavior
`getClassloader` should return null, when `servletContext` is `null` and the request should be "traced"

## Debug logs
None

### PR DESCRIPTION
**Pull request by tobiasstadler**
## What does this PR do?
Fixes #1860

## Checklist
- [ ] This is an enhancement of existing features, or a new feature in existing plugins
  - [ ] I have updated [CHANGELOG.asciidoc](https://github.com/elastic/apm-agent-java/blob/master/CHANGELOG.asciidoc)
  - [ ] I have added tests that prove my fix is effective or that my feature works
  - [ ] Added an API method or config option? Document in which version this will be introduced
  - [ ] I have made corresponding changes to the documentation
- [x] This is a bugfix
  - [x] I have updated [CHANGELOG.asciidoc](https://github.com/elastic/apm-agent-java/blob/master/CHANGELOG.asciidoc)
  - [ ] I have added tests that would fail without this fix
- [ ] This is a new plugin
  - [ ] I have updated [CHANGELOG.asciidoc](https://github.com/elastic/apm-agent-java/blob/master/CHANGELOG.asciidoc)
  - [ ] My code follows the [style guidelines of this project](https://github.com/elastic/apm-agent-java/blob/master/CONTRIBUTING.md#java-language-formatting-guidelines)
  - [ ] I have made corresponding changes to the documentation
  - [ ] I have added tests that prove my fix is effective or that my feature works
  - [ ] New and existing [**unit** tests](https://github.com/elastic/apm-agent-java/blob/master/CONTRIBUTING.md#testing) pass locally with my changes
  - [ ] I have updated [supported-technologies.asciidoc](https://github.com/elastic/apm-agent-java/blob/master/docs/supported-technologies.asciidoc)
  - [ ] Added an API method or config option? Document in which version this will be introduced
  - [ ] Added an instrumentation plugin? Describe how you made sure that old, non-supported versions are not instrumented by accident.
- [ ] This is something else
  - [ ] I have updated [CHANGELOG.asciidoc](https://github.com/elastic/apm-agent-java/blob/master/CHANGELOG.asciidoc)

### CODE REVIEW

File: apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/agent/servlet/helper/ServletTransactionCreationHelper.java [hunk 1]
```diff
@@ -85,6 +85,10 @@ private boolean isExcluded(String servletPath, @Nullable String pathInfo, @Nulla
 
     @Nullable
     public ClassLoader getClassloader(ServletContext servletContext){
```
Comment (kananindzya) [inline apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/agent/servlet/helper/ServletTransactionCreationHelper.java:87]: add please `@Nullable` annotation on param

Comment (tobiasstadler) [reply apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/agent/servlet/helper/ServletTransactionCreationHelper.java]: Done

**Comment by tobiasstadler** [comment]
@kananindzya Anything else I need to change?

File: apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/agent/servlet/helper/ServletTransactionCreationHelper.java [hunk 2]
```diff
@@ -78,7 +78,11 @@ private boolean isExcluded(String servletPath, @Nullable String pathInfo, @Nulla
     }
 
     @Nullable
-    public ClassLoader getClassloader(ServletContext servletContext){
+    public ClassLoader getClassloader(@Nullable ServletContext servletContext){
+        if(servletContext == null) {
```
Comment (felixbarny) [inline apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/agent/servlet/helper/ServletTransactionCreationHelper.java:82]:
```suggestion
        if (servletContext == null) {
```

**Review by felixbarny — APPROVED** [review]

**Comment by tobiasstadler** [comment]
Thank You!

**Pull request merged**